README.md

```diff
@@ -10,6 +10,17 @@ This library provides unofficial Go clients for [OpenAI API](https://platform.op
 * DALL·E 2
 * Whisper
 
+### Getting an OpenAI API Key:
```

Comment:
Great idea to put this into the README, thank you! I suggest we put it just before `Other examples:` — installation is basic usage is probably more important to keep on the top of the README

Reply:
@sashabaranov Thanks for review! I fixed it 👍